templates/inventory.ini.tpl

```diff
@@ -1,7 +1,20 @@
-[emqx]
+[emqx5]
+%{ if emqx_version_family == 5 ~}
```

Comment:
Q: should this and other occurrences in this file be `{% ...`?

Also, is `... ~}` intentional as well?  I couldn't find what that does.

Reply:
Maybe this is not Jinja2?

Reply:
That's right, this one is [terraform template](https://developer.hashicorp.com/terraform/language/functions/templatefile).
Tilda at the end is [whitespace stripping](https://developer.hashicorp.com/terraform/language/expressions/strings#whitespace-stripping).